src/components/InventoryHostStaleness/__test__/InventoryHostStaleness.test.js

```diff
@@ -47,12 +47,6 @@ describe('Table Renders', () => {
 
     await userEvent.click(screen.getByRole('button', { name: 'Edit' }));
 
-    expect(
-      screen.getAllByRole('button', { name: /options menu/i })
-    ).toHaveLength(3);
-    screen
-      .getAllByRole('button', { name: /options menu/i })
-      .forEach((button) => expect(button).toBeEnabled());
```

Comment:
Why was this needed to be removed?

Reply:
This shouldnt have been sent up, the names arent there anymore but I just find the buttons a different way now.